src/index.ts

```diff
@@ -7,7 +7,7 @@ import { Argument, LivePreviewProps, SubscribeCallback, TagAttributes } from './
 
 interface ContentfulLivePreviewInitConfig {
   debugMode?: boolean;
-  enableFieldTagging?: boolean;
+  enableInspectorMode?: boolean;
```

Comment:
This is a breaking change. Is it intentional?

Reply:
yes (was only brought in recently so low risk)